doc/src/manual/command-line-interface.md

```diff
@@ -101,6 +101,7 @@ The following is a complete list of command-line switches available when launchi
 |`--startup-file={yes*\|no}`            |Load `JULIA_DEPOT_PATH/config/startup.jl`; if `JULIA_DEPOT_PATH` environment variable is unset, load `~/.julia/config/startup.jl`|
 |`--handle-signals={yes*\|no}`          |Enable or disable Julia's default signal handlers|
 |`--sysimage-native-code={yes*\|no}`    |Use native code from system image if available|
+|`--pkgimage-native-code={yes*|no}`     |Use native code from package images if available ($)|
```

Comment:
```suggestion
|`--pkgimage-native-code={yes*\|no}`     |Use native code from package images if available ($)|
```
Need to escape the pipe character, otherwise Markdown will make a new column